src/aslm/model/devices/daq/daq_ni.py

```
@@ -173,11 +183,14 @@ def restart_analog_task_callback_func(task):
             Callback function
         """
 
+        # self.asi_stage = self.model.active_microscope.stages[self.axis]
         def callback_func(task_handle, status, callback_data):
             try:
+                logger.info("Analog Tasks Restarted")
                 task.stop()
                 task.start()
-            except Exception:
+            except:
```

except Exception is better than just a bare except.